pytorch_lightning/trainer/callback_config.py

```diff
@@ -95,3 +96,15 @@ def configure_early_stopping(self, early_stop_callback):
         else:
             self.early_stop_callback = early_stop_callback
             self.enable_early_stop = True
+
+    def check_callback_config(self):
+        for callback in self.callbacks:
```

Comment:
what about rather just filter and drop the wrong one meaning having [CB, CM, ES, CB] returns [CB, CB] and raise warnings that [CM, ES] was removed...